bin/spec

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why does this go away? It would be nice to have a global `spec` command that runs the Node and Go tests.